08-CMSIS/cmsis/STM32F429/cmsis.h

```diff
@@ -0,0 +1,38 @@
+/* mbed Microcontroller Library
```

Comment:
We should avoid copying CMSIS headers into the directory `08-CMSIS`. Instead, decoupling CMSIS implementations from `08-CMSIS` prevents from extra maintenance.